sdk/storage/azure-storage-queue/azure/storage/queue/_shared/base_client.py

```diff
@@ -320,11 +325,11 @@ def __exit__(self, *args):  # pylint: disable=arguments-differ
         pass
 
 
-def format_shared_key_credential(account, credential):
+def format_shared_key_credential(account_name, credential):
```

Comment:
we should rename this to `_format_shared_key_credential` . It's not exposed so rather make it private

Reply:
done!